### ISSUE TITLE
[Proposal] Add more than one icon at PWA install

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [w3c/manifest#841]
One of the feature that native apps have, but PWAs haven't at this moment, is adding more than one icon in the launcher and/or home screen pointing to different URLs (within the same host) at install.

I see this as extension to shortcuts — `separate` key (the name of key is draft):

```json
{
  "shortcuts": [
    {
      "name": "Play Later",
      "description": "View the list of podcasts you saved for later",
      "url": "/play-later",
      "icons": [
        {
          "src": "/icons/play-later.svg",
          "type": "image/svg+xml",
          "purpose": "any"
        }
      ]
    },
    {
      "name": "Subscriptions",
      "description": "View the list of podcasts you listen to",
      "url": "/subscriptions?sort=desc",
      "separate": true
    }
  ]
}
```

**Comment by NotWoods** [comment]
Currently the user has the option to drag out a shortcut into its own icon on Android. I don't see many apps publishing multiple icons without using this method. Do you have any examples where shortcuts are insufficient?

**Comment by FluorescentHallucinogen** [comment]
Yes, of course, user can drag out an item from shortcut menu into its own separate icon on Android, but it requires additional steps and not intuitive (obvious) for some users. Some users don't know about shortcuts at all.

**Comment by FluorescentHallucinogen** [comment]
As a use-case, I have an app for 360° photos in equirectangular-panoramic (equirect-pano) format. This app create 2 icons at install time, one is "Camera" (to take photos in this format) and the second for "Gallery" (to view photos in this format). The second app is Facebook. One icon is "Facebook" (for social network) and the second "Facebook Chat" (for messenger).

As you can see, creating more than one icon at install is useful for apps that can be splitted to two or more separate individual apps, but they are connected and should be used together. Installing 2 apps from at once is inconvenient. Shortcut menu items is not intuitive (obvious) for some users.

This feature is very useful for super-apps (apps-in-app model) PWA (see https://www.maketecheasier.com/what-are-super-apps/ and https://techcrunch.com/2019/09/19/the-emergence-of-the-super-apps-in-latin-america/ for more info about “super apps”). :wink:

**Comment by FluorescentHallucinogen** [comment]
@NekR, cc.

**Comment by aarongustafson** [comment]
This feels related to the idea of "projections" I mentioned at TPAC. I posted [a bit about it on WICG too](https://discourse.wicg.io/t/noodling-on-an-idea-projections-for-web-apps/3900).

**Comment by marcoscaceres** [comment]
closing due to lack of implementer interest.
